lang/python27/Portfile

```diff
@@ -90,7 +90,7 @@ subport ${name}-bootstrap {
     depends_extract     port:xz-bootstrap
     depends_skip_archcheck-append   xz-bootstrap
     extract.suffix      .tar.xz
-    extract.cmd         ${prefix}/bin/xz
+    extract.cmd         ${stdprefix}/libexec/xz-bootstrap/bin/xz
```

Comment:
this looks wrong

Reply:
@kencu nope, it doesn't. 20 line above it redefined `prefix` to `${prefix}/libexec/libcxx-bootstrap`

Reply:
stdprefix? let me see what is going on…